src/js/controllers/index.js

```diff
@@ -936,15 +939,14 @@ angular.module('copayApp.controllers').controller('indexController', function($r
     });
   }
 
-  self.showAllHistory = function() {
-    self.historyShowShowAll = false;
-    self.historyRendering = true;
+  self.showMore = function() {
     $timeout(function() {
-      $rootScope.$apply();
-      $timeout(function() {
-        self.historyRendering = false;
-        self.txHistory = self.completeHistory;
-      }, 100);
+      self.txHistory = self.txHistory.concat(self.completeHistory.slice(self.skipTxHistory, self.nextTxHistory));
```

Comment:
Please remove skipTxHistory and only use nextTxHistory.
and
   self.txHistory = self.completeHistory.slice(0, self.nextTxHistory);